easybuild/tools/modules.py

```diff
@@ -1106,7 +1107,7 @@ class Lmod(ModulesTool):
     """Interface to Lmod."""
     COMMAND = 'lmod'
     COMMAND_ENVIRONMENT = 'LMOD_CMD'
-    REQ_VERSION = '5.8'
+    REQ_VERSION = '6.6.3'
```

Comment:
note: Lmod 6.0 or newer is needed to pick up on `.modulerc` files in module file subdirectory.

Older Lmod 6.x versions have bugs that make it incompatible with EasyBuild in other (unrelated ways), hence the more recent 6.x being required.

Lmod 6.6.3 is the latest 6.x release, which was released Nov 13th 2016 (close to 2 years old), so I don't expect this to be a big problem.

Also, this way EasyBuild is still compatible with both Lmod 6.x & 7.x...